environment.yml

```diff
@@ -19,5 +19,4 @@ dependencies:
   - extension-helpers>=0.1
   - ginga>=3.0
   - linetools>=0.3
-  - PySide2>=5.0
```

Comment:
this is being removed from here and letting the dependencies that require it pull it in, correct?

Reply:
PySide2 was actually duplicated in environment.yml (might have been my fault a while back oops) so I'm just removing the duplicated entry

Reply:
Was probably my fault (via a merge).